rca/project_styleguide/templates/patterns/molecules/profile-detail/profile-detail.html

```diff
@@ -21,9 +21,11 @@
 
     <div class="profile-detail__content">
         <div class="profile-detail__overview">
-            <div class="profile-detail__header">
-                <h3 class="profile-detail__heading heading heading--five">{{ item.first_name }} {{ item.surname }}</h3>
-            </div>
+            {% if item.first_name %}
```

Comment:
should this test for both `item.first_name` and `item.surname`? or one or the other? I guess you don't want a surname on its own.

Reply:
I thought that as well, but i think that it's a safe assumption that they wouldn't want to just show the surname